CMakeLists.txt

```
@@ -99,6 +99,8 @@ endif()
 ## Pull in external support as needed
 ###############################################################################
 
+set (CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH};${CMAKE_CURRENT_SOURCE_DIR}/contrib)
```

I think the modern suggest is to use [`list(APPEND <list> [<element>...])`](https://cmake.org/cmake/help/latest/command/list.html#append)

```suggestion
list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/contrib)
```

---

Fixed